api/nakadi-event-bus-api.yaml

```diff
@@ -1217,6 +1221,7 @@ definitions:
       - name
       - category
       - owning_application
+      - enrichment_strategies
```

Comment:
It is. Actually, this field is required in the case of business and data change events. I don't know how to get rid of these exceptions. This category concept is really hard to deal with.